.github/scripts/hnctl.sh

```diff
@@ -1,20 +0,0 @@
-#!/bin/bash
```

Comment:
Mind explaining why this is no longer needed? Does Node 18 somehow take care of this automatically?

Reply:
The removed script installs a wrapper around hardhat as a npm package. This only slows down the CI run as we already have hardhat installed so we can just run it in the background with the npm script we have in the contracts folder. To shut it down we can use 'kill' coupled with 'lsof' to find the process id.

Reply:
hmm just did a quick check—the hnctl script didn't actually slow us down by much (11-13 sec vs 10 sec). Plus, it had port checking, which seems safer. What if we stuck to dedicated CI scripts for better readability?